test/Microsoft.Extensions.Logging.Testing.Tests/project.json

```diff
@@ -1,23 +1,26 @@
 {
-    "version": "1.0.0-*",
```

Comment:
We've been using 2 spaces in all other places. E.g. 
https://github.com/aspnet/Logging/blob/dev/src/Microsoft.Extensions.Logging/project.json
https://github.com/aspnet/Mvc/blob/dev/src/Microsoft.AspNet.Mvc/project.json